src/ucp/api/ucp_def.h

```diff
@@ -869,4 +869,15 @@ typedef struct {
 
 } ucp_transports_t;
 
+
+/**
+ * @ingroup UCP_COMM
+ * @brief Opaque descriptor list handle from @ref ucp_gpu_mem_list_create.
+ *
+ * This handle is opaque from host point of view. It is to be used from a GPU
+ * kernel using device specific functions.
```

Comment:
Maybe something like this instead of the second sentence:
`It is used by GPU-specific functions.`

Because `from GPU kernel`, and `using device specific functions` looks like the same thing said in two ways.